content/en/docs/event-sources/gvisor/_index.md

```diff
@@ -0,0 +1,188 @@
+
+Whenever a syscall gets executed inside the sandboxed application, the Sentry will manage it as usual, plus it will send a message to Falco through the UDS. Messages are serialized through Protocol Buffers so that gVisor and Falco can communicate even if they are written in different programming languages.
+
+Once a message related to a syscall is received, Falco unpacks it and creates the corresponding event in a way that is consumable by our libraries. This way, it is possible to update necessary state information and trigger Falco rules whenever a match occurs!
+
+![](images/gvisor-02.png)
+
+## Setup gVisor Docker sandbox monitoring with Falco
+
+First, [install **Falco**](https://falco.org/docs/getting-started/installation/) 0.33.1 or above and [install the **gVisor runsc tool**](https://gvisor.dev/docs/user_guide/install/) 20221122.0 or above.
+
+You can check the version by running:
+```bash
+falco --version
+```
+... which needs to report 0.33.1 or above and:
+```bash
+runsc --version
+```
+... which needs to report `release-20221122.0` or above.
+
+gVisor needs to be configured to send events to Falco. Generate the appropriate configuration file:
+
+```bash
+$ falco --gvisor-generate-config > /tmp/runsc_falco_config.json
+$ sudo mv /tmp/runsc_falco_config.json /etc/docker/runsc_falco_config.json
+
+# Don't forget to protect this configuration
+$ sudo chmod 640 /etc/docker/runsc_falco_config.json
+```
+
+The easiest way to run a gVisor sandbox is by using Docker. For this reason, you need to first [configure Docker to work with gVisor via `runsc install`](https://gvisor.dev/docs/user_guide/quick_start/docker/), and then we're going to update the `runsc` pod init config configuration for our Docker containers:
+
+```bash
+$ sudo -e /etc/docker/daemon.json
+```
+
+Then, add the `runtimeArgs` key with the `--pod-init-config=` parameter like in the example below:
+
+```json
+{
+    "runtimes": {
+        "runsc": {
+            "path": "/usr/local/bin/runsc",
+
+--- Do not forget the comma at the end of the previous line. ---
+--- Add the following config, not these instructions.        ---
+
+            "runtimeArgs": [
+                "--pod-init-config=/etc/docker/runsc_falco_config.json"
+
+--- End of added config.                                     ---
+--- Have I told you not to include these instructions.       ---
+
+            ]
+        }
+    }
+}
+```
+
+Then, restart the Docker daemon to let it use the new configuration:
+
+```bash
+$ sudo systemctl restart docker
+```
+
+## Runtime detection in action
+
+Now it's time to put everything together and see how to use Falco to monitor gVisor sandboxes.
+To start monitoring gVisor sandboxes, you can use the `-g` or `--gvisor-config` options, passing the path to the pod init config. Falco uses that config file for two main reasons:
```

Comment:
I believe these options are deprecated and have been replaced by `-o ...` or the Falco configuration file, see the documentation about the modern eBPF: https://falco.org/docs/install-operate/running/#modern-ebpf